esx_service/cli/vmdkops_admin.py

```diff
@@ -453,20 +455,23 @@ def ls(args):
     If args.l is True then show all metadata in a table.
     If args.c is not empty only display columns given in args.c (implies -l).
     """
+    tenant_reg = '*'
+    if args.tenant:
+        tenant_reg = args.tenant
+
     if args.c:
-        (header, rows) = ls_dash_c(args.c)
+        (header, rows) = ls_dash_c(args.c, tenant_reg)
     else:
         header = all_ls_headers()
-        rows = generate_ls_rows()
-    if args.tenant:
-        print("TBD: print volumes for the given tenant ", args.tenant)  # TODO
+        rows = generate_ls_rows(tenant_reg)
+   
     print(cli_table.create(header, rows))
 
 
-def ls_dash_c(columns):
+def ls_dash_c(columns, tenant_reg):
     """ Return only the columns requested in the format required for table construction """
     all_headers = all_ls_headers()
-    all_rows = generate_ls_rows()
+    all_rows = generate_ls_rows(tenant_reg)
```

Comment:
Nit, if the tenant doesn't exist or no volumes created via the tenant then do we want to print even the header or a message to the effect of no volumes found. Else the user won't know whether the command is not finding the volumes or there really aren't any volumes.